app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditLoginItems.kt

```diff
@@ -518,13 +520,30 @@ private fun PasswordRow(
 private fun PasskeyField(
     creationDateTime: Text,
     modifier: Modifier = Modifier,
+    canViewPassword: Boolean,
+    loginItemTypeHandlers: VaultAddEditLoginTypeHandlers,
 ) {
-    BitwardenTextField(
+    BitwardenTextFieldWithActions(
         label = stringResource(id = R.string.passkey),
         value = creationDateTime.invoke(),
         onValueChange = { },
         readOnly = true,
         singleLine = true,
         modifier = modifier,
+        actions = {
+            if (canViewPassword) {
```

Comment:
`canViewPassword` seems misleading. It's controlling visibility of the "Remove" button, not a password field. Can we rename this to `canRemovePasskey`, or something similar?